### DISCUSSION TITLE
As a father: what I wish everyone knew about Autism – Tiberriver256

### DISCUSSION THREAD

**Discussion by giscus** [Tiberriver256/Tiberriver256.GitHub.io#15] [General]
# As a father: what I wish everyone knew about Autism – Tiberriver256

As a father, I wish everyone knew that each autistic child is different

https://tiberriver256.github.io/autism/as-a-father-what-i-wish-people-knew-about-autism/

**Comment by grwinklers** [comment]
While I have very little visibility to autism, it is awesome for for those who do to educate. Thanks Micah!
